docs/writing_plugins.rst

```diff
@@ -29,14 +29,64 @@ A plugin with a path helper function may look something like this:
 
 
     class MyPlugin(BasePlugin):
-        def path_helper(self, path, func, **kwargs):
+        def path_helper(self, path, func, parameters, **kwargs):
             """Path helper that parses docstrings for operations. Adds a
             ``func`` parameter to `apispec.APISpec.path`.
             """
             operations = load_operations_from_docstring(func.__doc__)
             return Path(path=path, operations=operations)
 
 
+All helpers take `**kwargs` arguments that are passed to `apispec.APISpec` methods by users and forwarded to plugins methods,
```

Comment:
Suggestion:

All plugin helpers must accept extra `**kwargs`, allowing custom plugins to define new arguments if required.